tools/harness-thci/OpenThread.py

```diff
@@ -551,6 +551,20 @@ def __setAddressfilterMode(self, mode):
         except Exception as e:
             ModuleHelper.WriteIntoDebugLogger('__setAddressFilterMode() Error: ' + str(e))
 
+    def __skipSeqNoIncrease(self):
+        """skip sequence number increase when recovering BBR Dataset from Network Data
+
+        Returns:
+            True: successful to set the behavior.
+            False: fail to set the behavior.
+        """
+        print('call __skipSeqNoIncrease()')
+        try:
+            cmd = 'bbr skipseqnoinc'
+            return self.__executeCommand(cmd)[-1] == 'Done'
+        except Exception as e:
+            ModuleHelper.WriteIntoDebugLogger('__setAddressFilterMode() Error: ' + str(e))
```

Comment:
```suggestion
        cmd = 'bbr skipseqnoinc'
        return self.__executeCommand(cmd)[-1] == 'Done'
```

suggest no catching Exception